src/pages/en/tutorial/1-setup/index.md

```diff
@@ -13,6 +13,39 @@ Now that you know what you're going to build, let's set up all the tools and acc
 
 Skip ahead to [Unit 2](/en/tutorial/2-pages/) if you already have a working development environment, and can [create](/en/install/auto/) and deploy the [Astro minimal starter template](https://github.com/withastro/astro/tree/main/examples/minimal) to the web on your own.
 
+:::tip[using stackblitz]
+
+Want to complete this tutorial in an online code editor instead?
+<details>
+<summary>Follow these instructions!</summary>
```

Comment:
```suggestion
<summary>Follow these instructions, then go directly to Unit 2!</summary>
```
I'm thinking it might help to be a little explicit here?